opm-core/Backups.rst

```diff
@@ -0,0 +1,22 @@
+Backups
```

Comment:
I would prefer to mention backup/restore than only backup, especially since only the restore part is problematic.

Reply:
Done